src/Symfony/Component/DependencyInjection/Compiler/AttributeAutoconfigurationPass.php

```diff
@@ -167,4 +167,23 @@ protected function processValue(mixed $value, bool $isRoot = false): mixed
 
         return parent::processValue($value, $isRoot);
     }
+
+    /**
+     * Looks for a configurator for the given attribute name in the given configurators array.
+     * If no configurator is found, the parent class is checked.
+     * If no configurator is found, null is returned.
+     * The result is cached in the configurators array passed by reference.
+     */
```

Comment:
I may remove or reword this comment.

Reply:
it could tell a bit more about the "why" than the "how",  and be slightly reworded indeed